schemas/GO/GO_edges_consider.yaml

```diff
@@ -0,0 +1,27 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+title: GO_edges_consider
+type: edge
+description: A entry for consider edges in the Gene Ontology (GO) hierarchy
+properties:
+  id:
+    type: string
+    description: GO id
+    examples:
+    - GO:0000136__GO:0031501__consider
```

Comment:
If you're separating edges into collections the edge type isn't really needed as part of the id any more